driver/network_driver.go

```diff
@@ -65,13 +69,29 @@ func NewNetworkDriver(client *datastoreClient.Client) network.Driver {
 		ifPrefix:         IFPrefix,
 		DummyIPV4Nexthop: "169.254.1.1",
 
+		vethMTU: 1500,
+
 		// default: enabled, disable by setting env key to false (case insensitive)
 		createProfiles: !strings.EqualFold(os.Getenv(CREATE_PROFILES_ENVKEY), "false"),
 
 		// default: disabled, enable by setting env key to true (case insensitive)
 		labelEndpoints: strings.EqualFold(os.Getenv(LABEL_ENDPOINTS_ENVKEY), "true"),
 	}
 
+	// Check if MTU environment variable is given, parse into uint16
+	// and override the default in the NetworkDriver.
+	if mtuStr, ok := os.LookupEnv(VETH_MTU_ENVKEY); ok {
+		mtu, err := strconv.ParseUint(mtuStr, 10, 16)
+		if err != nil {
+			log.Fatalf("Failed to parse %v '%v' into uint16: %v",
+				VETH_MTU_ENVKEY, mtuStr, err)
+		}
+
+		driver.vethMTU = uint16(mtu)
+
+		log.Info("calico-net veth MTU set to ", mtu)
```

Comment:
Nit: we generally prefer logs which use `WithField` (though I know a lot of the logs in this file don't use that approach!)

e.g.

```
log.WithField("mtu", mtu).Info("Parsed veth MTU")
```